Library/Homebrew/cask/audit.rb

```diff
@@ -482,13 +482,25 @@ def audit_signing
       odebug "Auditing signing"
 
       extract_artifacts do |artifacts, tmpdir|
+        is_container = artifacts.any? { |a| a.is_a?(Artifact::App) || a.is_a?(Artifact::Pkg) }
```

Comment:
If there's a standalone binary (typically a shell script) included in or with an application bundle, we can skip auditing it.

Many times those scripts aren't signed, but if the application is we'll consider it trusted.  If the app isn't trusted, it will fail the normal check.

Reply:
Makes sense!